itou/templates/siaes/edit_job_description.html

```diff
@@ -37,6 +37,10 @@ <h1 class="text-center mb-5">Informations générales<br> de ma fiche de poste</
             {% bootstrap_field form.location_label %}
             {% bootstrap_field form.location_code %}
 
+            {% if siae.is_opcs %}
+                {% bootstrap_field form.market_context_description %}
+            {% endif %}
```

Comment:
Je comprends mieux pourquoi tu disais « ça vient un commit trop tôt » ! :)
Ce champ ne devrait pas être conditionnel à cet endroit mais dans le form car c'est dans ce dernier que se concentre toute la logique des formulaires. Les gabarits sont aveugles, ils ne devraient pas contenir de logique métier.

Je serais donc vraiment d'avis de gérer l'affichage de ce champs ainsi que de `is_qpv_mandatory` dans le formulaire et non ici.
Tu as quelques exemples dans [les formulaires des approvals](https://github.com/betagouv/itou/blob/master/itou/www/approvals_views/forms.py#L138).

> In much the same way that a Django model describes the logical structure of an object, its behavior, and the way its parts are represented to us, a [Form](https://docs.djangoproject.com/en/4.0/ref/forms/api/#django.forms.Form) class describes a form and determines how it works and appears.
> In a similar way that a model class’s fields map to database fields, a form class’s fields map to HTML form <input> elements. 

[Doc de Django](https://docs.djangoproject.com/en/4.0/topics/forms/#the-django-form-class)

Reply:
Alors moi, imagine. Je prends le point et sors le métier (test opcs) du gabarit.

Reply:
Cf mon commentaire dans slack, je n'ai pas de réponse évidente. A toi de tester pour voir le meilleur compromis entre nombre de lignes de code et simplicité/niveau de piège desdites lignes.